opae-libs/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ project(opae-libs)
 
 set(OPAE_VERSION_MAJOR    2 CACHE STRING "OPAE major version" FORCE)
 set(OPAE_VERSION_MINOR    0 CACHE STRING "OPAE minor version" FORCE)
-set(OPAE_VERSION_REVISION 0 CACHE STRING "OPAE revision version" FORCE)
+set(OPAE_VERSION_REVISION 1 CACHE STRING "OPAE revision version" FORCE)
```

Comment:
Merge OPAE-libs version change PR to Master

Reply:
@tswhison does a new branch/PR with a new `git subtree` operation (now done on opae-libs `master` instead of opae-libs `release/2.0.1-1`) need to be done to pull in the version change to opae-sdk `master`?

Reply:
yes, it does.

Reply:
Thanks @tswhison. Closing PR, will open new PR shortly.